avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
+
```

Comment:
> What I don't understand is why was I unable to reproduce it when I added time.sleep(10) in the _bootstrap(). Were you able to manually reproduce it that way?

No, the only reproducer I have is travis.

> Also all of the failures were after the early_status was pushed, therefor the start_logging was already executed. That's why I'm wondering, whether it fixes the issue.

`proc.start()` is executed before `queue.put(early_state)`. If the subprocess is faster than the parent process, we can reach the `_bootstrap`'s  `SystemExit` handler before `queue.put(early_state)` is executed. That's my bet.